user_guide_src/source/incoming/controllers.rst

```diff
@@ -26,13 +26,14 @@ Let's try it: Hello World!
 ==========================
 
 Let's create a simple controller so you can see it in action. Using your text editor, create a file called Helloworld.php,
-and put the following code in it::
+and put the following code in it. You will notice that the Helloworld Controller is extending the BaseController you can
+also extend the Controller if you do not need the functionality of the BaseController.::
```

Comment:
I think it would be nice to explain here the functionality of the `BaseController` - since we mention it and advise users to use it. Maybe just copy-paste the description from it?